src/viam/sdk/rpc/dial.cpp

```diff
@@ -88,8 +96,12 @@ const boost::optional<DialOptions>& Options::dial_options() const {
     return dial_options_;
 }
 
+// Deprecated! use Credentials(std::string payload, std::string type) instead
```

Comment:
If we want users to see this, the comment should be in the `.hpp` file with the `///` to indicate that it's a doc comment. Probably we should amend the language a bit as well (`Deprecated!` is a little informal for a doc site)

Reply:
hpp only. In general I think of cpp comments as for people who want to understand how the code works (but there's nothing interesting about how this works, it's a totally straightforward constructor), and hpp comments are for how to use the code.